bundle/language_sw.properties

```diff
@@ -489,6 +489,7 @@ angal.lab.pleaseselectapatient
 angal.lab.positive                                                               = sahihi
 angal.lab.printreport                                                            = Piga Chapa repoti
 angal.lab.printtable                                                             = Piga chapa mraba
+angal.lab.print																	 = Piga Chapa
```

Comment:
convertion space->tabs happening here

Reply:
Fixed with the push c9b6c019656dba1594baf288c36cf9d381f4130d